package.json

```diff
@@ -49,45 +49,45 @@
     }
   },
   "peerDependencies": {
-    "webpack": "^4.0.0"
+    "webpack": "^4.43.0"
```

Comment:
I know it is technically a minor version increase here, but it is a big jump. Did we check to make sure everything is working well with the webpack in downstream projects?

Reply:
Yes. This has been tested, but I should probably keep the peer dep at 4.0. Good callout!